### PR TITLE
Link homepage, Twitter and YouTube

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: documentation/modules/ROOT/pages/04-references.adoc [hunk 1]
```diff
@@ -1,8 +1,11 @@
 = References
 include::_attributes.adoc[]
 
-You can find more detail about Thoth's security recommendations for containers and other Thoth functionalities in the following articles:
+You can find more detail about Thoth's security recommendations and other Thoth functionalities in the following articles:
```
Comment (mayaCostantini) [inline documentation/modules/ROOT/pages/04-references.adoc:4]:
```suggestion
You can find more detail about Thoth's security recommendations and other Thoth functionalities on our social media accounts and in the following articles:
```

**Review by mayaCostantini — COMMENTED** [review]
Just one comment otherwise lgtm, thanks :rocket: :100:

File: documentation/modules/ROOT/pages/04-references.adoc (hunk 1 above)
Comment (fridex) [reply documentation/modules/ROOT/pages/04-references.adoc]: Thanks! 👍🏻

**Review by mayaCostantini — APPROVED** [review]
Thanks! :100:

**Pull request merged**